ansible/roles/mysql-user/tasks/main.yml

```diff
@@ -4,6 +4,8 @@
     src: "resources/my.cnf"
     dest: "/etc/my.cnf"
   become: true
+  retries: 10
```

Comment:
If the purpose of this is to wait for mysql vm to come up and be available on dns, I do not think this is the right place for this.
if candlepin.yml is shuffled at some point, this may not be the first step after the vm restart.
I'd recommend something at that location to verify everything is accessible before we continue.